.claude/commands/dfetch-add.md

```
@@ -0,0 +1,82 @@
+Add a new project to `dfetch.yaml` and fetch it.
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)